ccd/models/lasso.py

```diff
@@ -29,17 +29,26 @@ def coefficient_matrix(dates, avg_days_yr, num_coefficients):
 
     matrix = np.zeros(shape=(len(dates), 7), order='F')
 
+    # lookup optimizations
+    # Before optimization - 12.53% of total runtime
+    # After optimization  - 10.57% of total runtime
+    w12 = w * dates
```

Comment:
If the '12', '34', and '56' refer to the coefficients, the intercept and slope are really the first two.
The scipy lasso implementation handles the intercept, but line 41 is for the slope.